### ISSUE TITLE
[HW3 Testing] Coverage의 meaning 관련 질문 

### ISSUE THREAD

**Issue by diffzerg** [swsnu/swppfall2022#142]
조교님 안녕하세요, Testing의 점수에 들어가는 Coverage란 % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s 
중에서 Stmts를 의미하는 것인가요?

**Comment by jaewooMaeng** [comment]
구체적인 채점 기준을 말씀드릴 수는 없지만, coverage를 찍었을 때 보실 수 있는 항목들을 종합적으로 고려합니다. stmts 외의 값도 채점 기준에 포함됩니다.

**Comment by diffzerg** [comment]
넵 확인 감사합니다!

**Comment by jaeyoungleeee** [comment]
@jaewooMaeng 
안녕하세요 조교님!
관련하여 추가적인 질문이 있습니다!

말씀해주신 내용을 고려하면, jest test 를 돌렸을 때 나오는 지표 "이외"의 평가요소는 없다고 생각해도 될까요?

감사합니다!

**Comment by jaewooMaeng** [comment]
@jaeyoungleeee 맞습니다. test를 돌렸을 때 나오는 지표만 신경써주시면 됩니다. 아시는 것처럼 feature의 구현 점수에 가중치가 곱해집니다.
